commands/Tools/todo.js

```diff
@@ -0,0 +1,54 @@
+/*
+To your main server file, add:
+Client.defaultUserSchema
+  .add("TODOs", "any", {array: true, default: [], configurable: true});
+*/
+
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			enabled: true,
+			runIn: ['dm'],
+			requiredPermissions: [],
+			requiredSettings: [],
+			aliases: [],
+			autoAliases: true,
+			bucket: 1,
+			cooldown: 0,
+			promptLimit: 0,
+			promptTime: 30000,
+			guarded: false,
+			permissionLevel: 0,
+			description: 'add|remove|list user\'s TODOs through DM',
+			extendedHelp: 'No extended help available.',
+			usage: '<add|remove|list> (TODO:string) [content:...string]',
```

Comment:
```suggestion
			usage: '<add|remove|list:default> (TODO:string) [content:...string]',
```

So when you do `@bot todo`, it shows you the list. It's more intuitive than not defaulting.